benchmark/domain/domain-fn-args.js

```diff
@@ -12,14 +12,14 @@ var gargs = [1, 2, 3];
 
 function main(conf) {
 
-  var args, n = +conf.n;
```

Comment:
Yeah but `var` is not block scoped so I think this only create confusion for newbies. Can `const` be used instead?